packages/expo-video/src/VideoPlayer.types.ts

```diff
@@ -232,6 +231,25 @@ export declare class VideoPlayer extends SharedObject<VideoPlayerEvents> {
    */
   readonly isExternalPlaybackActive: boolean;
 
+  /**
+   * Determines the time that the actual position seeked to may precede or exceed the requested seek position.
+   *
+   * This property affects the precision of setting the [`currentTime`](#currenttime) property and the [`seekBy`](#seekbyseconds) method, on Android it also affects the accuracy of the scrubber from the default native controls.
+   *
+   * By default, the player seeks to the exact requested time.
+   *
+   * > If you are trying to optimize for scrubbing (many frequent seeks), also see [`ScrubbingModeOptions`](#scrubbingmodeoptions).
```

Comment:
```suggestion
   * > If you are trying to optimize for scrubbing (many frequent seeks), also see [`ScrubbingModeOptions`](#scrubbingmodeoptions-1).
```

Since the property and the type are both spelled the same.